installation/language/en-GB/joomla.ini

```
@@ -185,7 +206,7 @@ INSTL_ERROR_INITIALISE_SCHEMA="Can't initialise database schema."
 INSTL_FILE_UPLOADS="File Uploads"
 INSTL_GNU_GPL_LICENSE="GNU General Public License"
 INSTL_HELP_LINK="Help installing Joomla"
-INSTL_NOTICEYOUCANSTILLINSTALL="You can still continue the installation if you repair the permissions or you provide an FTP connection."
+INSTL_NOTICEYOUCANSTILLINSTALL="To successfully install Joomla, you have three choices:<ul><li>Repair permissions</li><li>Provide FTP credentials with write access to the folder or</li><li>Manually create the file at the end of the installation.</li></ul>"
```

```suggestion
INSTL_NOTICEYOUCANSTILLINSTALL="To install Joomla, you have three choices:<ul><li>Repair permissions</li><li>Provide FTP credentials below with write access to the folder or</li><li>Manually create the file at the end of the installation.</li></ul>"
```

Remove the redundant word successfully and add the word below as it might not be clear where to do that if the inputs are not in view